tests/spark/test_spark_model_export.py

```diff
@@ -350,7 +350,7 @@ def _transform(self, dataset):
     unsupported_pipeline = Pipeline(stages=[CustomTransformer()])
     unsupported_model = unsupported_pipeline.fit(spark_model_iris.spark_df)
 
-    with pytest.raises(mleap.MLeapSerializationException):
+    with pytest.raises(ValueError):
```

Comment:
Why do we expect `ValueError` instead of `MLeapSerializationException`?

Reply:
Good question, it's because we now attempt to write the Spark model ([here](https://github.com/mlflow/mlflow/pull/808/files#diff-47cfaaee925e76d2dc8ec604c2ef5f73R122)) before persisting the MLeap model ([here](https://github.com/mlflow/mlflow/pull/808/files#diff-47cfaaee925e76d2dc8ec604c2ef5f73R137)) - MLlib raises a ValueError when `spark_model.save()` on a Pipeline containing an unsupported Transformer, as is visible in this travis build: https://travis-ci.org/mlflow/mlflow/jobs/480164372

Reply:
Got it. I think it's because we changed the order of operations for `save_model`, which seems fine.